library/std/src/ffi/os_str.rs

```diff
@@ -1101,6 +1101,24 @@ impl<'a> From<Cow<'a, OsStr>> for OsString {
     }
 }
 
+#[stable(feature = "str_tryfrom_osstr_impl", since = "1.64.0")]
```

Comment:
Does this Rust version need to be updated before merging?

```suggestion
#[stable(feature = "str_tryfrom_osstr_impl", since = "CURRENT_RUSTC_VERSION")]
```